src/app/components/header-mobile/header-mobile.component.html

```diff
@@ -38,27 +38,27 @@
 @if(isMobileSideNavOpen){
 <div class="sidenav">
   <ul>
-    <li><button (click)="toggle()"><a routerLink="home">Création De Personnage</a></button></li>
+    <li><button (click)="isMobileSideNavOpen=false"><a routerLink="accueil">Création De Personnage</a></button></li>
```

Comment:
same here, you can put the click option inside the a if it triggers some special action (like closing the navbar menu)